acpi.adoc

```diff
@@ -64,9 +69,16 @@ objects.
 | [[acpi-tad]] AML_070 | Systems with a Real-Time Clock on an OS-managed bus (e.g. I2C, subject to arbitration issues due to access to the bus by the OS) MUST implement the Time and Alarm Device (TAD).
 2+| _Also see <<uefi-rtc, URT_020>>_.
 | AML_080 | Systems implementing a TAD must be functional without additional system-specific OS drivers.
+| [[acpi-irq-gsb]] AML_090 | PLIC and APLIC devices must support
+**_GSB** method. <<acpi-ids, See RVI ACPI IDs>>.
+| [[acpi-irq-dep]] AML_100 | Devices which use Global System
+Interrupts (GSI) must indicate the dependency on corresponding
+interrupt controller using **_DEP** method.
```

Comment:
We discussed this in PRS TG. This should be optional. Other OS may not have the same issue. Firmware doesn't need to specify _GSB and _DEP in that case. Isn't it ?

Reply:
Yeah, but FW will be common for different OSs. So I think even if one OS needs it, it will become mandatory for firmware. I was initially thinking keeping it bit open saying "Devices should indicate the dependency appropriately" without mentioning any ways like _DEP. But I guess that might cause confusion as well.

Reply:
Yeah that wouldn't be really actionable, because "indicating appropriately" sounds like being able to enumerate every crutch needed by every OS kernel out there :-).